### ISSUE TITLE
Add 'is_impossible' tag to annotated datasets

### ISSUE THREAD

**Issue by Rob192** [etalab/piaf#41]
## Problem description
We are using knowledge base formated per the squad v2 format, see https://etalab-ia.github.io/knowledge-base/piaf/howtos/integrate.html
When using data from Piaf Anno, the data is formated in v1 format

## Solution description
Add is impossible keys to dataset generated with piaf anno

**Comment by guillim** [comment]
Good point ! Anyone could seize this issue : it's a good "first-commit" on Piaf
